tests/integration/cartography/intel/github/test_teams.py

```diff
@@ -0,0 +1,114 @@
+from unittest.mock import patch
+
+import cartography.intel.github.teams
+from cartography.intel.github.teams import sync_github_teams
+from tests.data.github.teams import GH_TEAM_DATA
+from tests.data.github.teams import GH_TEAM_REPOS
+from tests.integration.cartography.intel.github.test_repos import _ensure_local_neo4j_has_test_data
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
+
+TEST_UPDATE_TAG = 123456789
+TEST_JOB_PARAMS = {'UPDATE_TAG': TEST_UPDATE_TAG}
+TEST_GITHUB_URL = "https://fake.github.net/graphql/"
+FAKE_API_KEY = 'asdf'
+
+
+@patch.object(cartography.intel.github.teams, '_get_team_repos', return_value=GH_TEAM_REPOS)
+@patch.object(cartography.intel.github.teams, 'get_teams', return_value=GH_TEAM_DATA)
+def test_sync_github_teams(mock_teams, mock_team_repos, neo4j_session):
+    # Arrange
+    _ensure_local_neo4j_has_test_data(neo4j_session)
```

Comment:
Non blocker: add one more test org to make sure teams aren't accidentally added to that other org.